network-api/networkapi/templates/donate/fragments/formassembly_body.html

```diff
@@ -89,7 +89,7 @@
                     donations via direct debit in Euros through Single European Payment Area
                     (SEPA) transfer.
                 {% endblocktrans %}
-            </span><br><br><div><b><a target="_blank" href="https://foundation.mozilla.org/?form=donate&amp;utm_medium=web&amp;utm_source=faq-page&amp;utm_campaign=23-Donor-Care&amp;utm_term=en"><u>{% trans "Make your donation online now" %}<br><br></u></a></b><i>{% trans "Still have questions? Contact us by submitting a case." %}</i><br></div></div></div></div>
+            </span><br><br><div><b><a href="?form=donate&amp;utm_medium=web&amp;utm_source=faq-page&amp;utm_campaign=23-Donor-Care&amp;utm_term=en"><u>{% trans "Make your donation online now" %}<br><br></u></a></b><i>{% trans "Still have questions? Contact us by submitting a case." %}</i><br></div></div></div></div>
```

Comment:
Updated this in form assembly to be a relative URL, that way, the donate form will open without leading the users to the foundation homepage on another tab!

Also updated revision number on both this file and `formassembly_head.html` to reflect new version number

Reply:
Hey @danielfmiranda , jsut wanted to confirm that it is okay to hardcoded`utm_term`  to `en`

Reply:
Thanks for the catch @mmmavis! After confirming with @miaatmozilla that this should be removed, I have updated the link in form assembly and also updated this file to remove `utm_term=en`